packages/auth/src/storage/keys.ts

```diff
@@ -0,0 +1,59 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+export enum CognitoUserPoolKey {
+	accessToken = 'accessToken',
+
+	refreshToken = 'refreshToken',
+
+	idToken = 'idToken',
+}
+
+export enum LegacyCognitoUserPoolKeys {
+	accessToken = 'accessToken',
+
+	refreshToken = 'refreshToken',
+
+	idToken = 'idToken',
+
+	lastAuthUser = 'LastAuthUser',
+
+	clockDrift = 'clockDrift',
+
+	userData = 'userData',
+}
+
+export enum CognitoDeviceKey {
+	deviceKey = 'deviceKey',
+
+	deviceGroupKey = 'deviceGroupKey',
+
+	randomPasswordKey = 'randomPasswordKey',
+}
+
+/**
+ * The key to get/set the auth token storage manager version.
+ */
+export const USER_POOL_MANAGER_VERSION_KEY = 'UserPoolManagerVersionKey';
```

Comment:
Nit: Do we need to include `Key` in the key ID since it's kind of implied? Ditto for `DEVICE_KEY_MANAGER_VERSION_KEY`

```suggestion
export const USER_POOL_MANAGER_VERSION_KEY = 'UserPoolManagerVersion';
```